manim/mobject/numbers.py

```diff
@@ -139,3 +140,18 @@ class Integer(DecimalNumber):
 
     def get_value(self):
         return int(np.round(super().get_value()))
+
+
+class Variable(VDict):
+    def __init__(self, var, label):
+        var_label = TexMobject(label + "=")
+        self.var_tracker = ValueTracker(var)
+        value_tex = (
+            DecimalNumber(self.var_tracker.get_value())
+            .add_updater(lambda v: v.set_value(self.var_tracker.get_value()))
+            .next_to(var_label, RIGHT)
+        )
+        VDict.__init__(self, {"label": var_label, "value": value_tex})
```

Comment:
Comment: I really like the idea of having some of the pre-defined classes inherit from VDict. However, I'm wondering if it's better to have `tracker["label"]`, or just `tracker.label`. The latter is more succinct and it doesn't need a VDict interface. What do you think?

Reply:
I am divided on this. It depends which one is more intuitive:
```python
on_screen_var["label"].set_color(RED)
```
OR
```python
on_screen_var.label.set_color(RED)
```
In the first one, I can see the "label" more binding to `on_screen_var`. In the second one, though it is obvious that `label` is not a method, it gives me a feel that label might be method (to label something) and somebody forgot the brackets. However, it would be great to have more opinion on this.

Reply:
Also, without `VDict`, I am not really sure, how will we get `ShowCreation(on_screen_var)` to work. We will have to invoke `ShowCreation` for the label as well as value.

Reply:
Oh good point. Yeah so this would have to be either a VGroup or a VMobject that has a couple of pre-defined submobjects. I would also like more opinion on the main point. Perhaps @Aathish04 ?

Reply:
To keep things as generic as possible, I'd rather it inherit from `VMobject`. The `on_screen_var` could be a submobject of said `VMobject`.